modules/ROOT/pages/rest-api-extensions.adoc

```diff
@@ -29,7 +29,7 @@ There is some additional prerequisites when creating a REST API extension:
 . Enter a package name, use to set the artifact *Group id*, for example: _com.company.rest.api_
 . Enter a *Project name*, for example _userInformationRestAPIExtension_
 . Click *Next*.
-. Enter the *pathTemplate* for this REST API extension, for example _userInformation_. This will be the access point of the API, and follows this pattern: `+{bonita_portal_context}/API/extension/userInformation+`.
+. Enter the *pathTemplate* for this REST API extension, for example _userInformation_. This will be the access point of the API, and follows this pattern: `+{bonita_context}/API/extension/userInformation+`.
 . As this REST API extension does not access business data you can safely uncheck "Add BDM dependencies" check box.
 . Define a *Permission* name for the extension (replace the default one), for example _read_user_information_. This is the name of the permission the users should have to be granted access to the extension (see xref:api-extensions.adoc#usage[REST API extensions usage]
```

Comment:
This is the name of the permission the users should have to be granted access to the extension => This will be the name of the permission the users should have to be granted in order to have access to the extension